storm-core/src/storm.thrift

```diff
@@ -243,6 +243,16 @@ struct SubmitOptions {
   2: optional Credentials creds;
 }
 
+enum NumErrorsChoice {
+  ALL,
+  NONE,
+  ONE
+}
+
+struct GetInfoOptions {
+  1: optional NumErrorsChoice num_err_choice;
```

Comment:
I hadn't thought about it that way, but I'm open to it.  Could you give an example of what you mean?